src/pages/map.tsx

```diff
@@ -0,0 +1,50 @@
+import {
+  Tab,
+  Tabs,
+  TabList,
+  TabPanel,
+  TabPanels,
+  Flex,
+  Heading,
+} from '@chakra-ui/react';
+import React from 'react';
+import { sectionSpacing } from 'theme/metrics';
+
+export default function Page() {
+
+    return (
+    <Flex direction="column" gap={sectionSpacing}>
```

Comment:
Why do you need a Flex here? Actual content in a tab should be wrapped into <TabPanel>, no?